src/prometheus_alert_rules/pressure.rules

```diff
@@ -0,0 +1,33 @@
+groups:
+- name: HostPressure
+  rules:
+
+  # Alert for host cpu pressure - high instantanoues cpu waiting time
+  - alert: HostHighCpuWaitingTime
+    expr: irate(node_pressure_cpu_waiting_seconds_total[5m]) > 0.9
+    for: 5m
+    labels:
+      severity: info
+    annotations:
+      summary: Host processes spent too much time waiting for CPU resources (instance {{ $labels.instance }})
+      description: The instantanoues time that the processes spent on waiting for CPU resource is too high. This might indicates that the server is under high CPU pressure.\n  VALUE = {{ $value | printf "%.2f" }}\n  LABELS = {{ $labels }}
+
+  # Alert for host io pressure - high instantanoues io waiting time
+  - alert: HostHighIOWaitingTime
+    expr: irate(node_pressure_io_waiting_seconds_total[5m]) > 0.5
+    for: 5m
+    labels:
+      severity: info
+    annotations:
+      summary: Host processes spent too much time waiting due to I/O congestion (instance {{ $labels.instance }})
+      description: The instantanoues time that the processes spent on waiting for I/O is too high. This might indicates that the server is under high I/O pressure.\n  VALUE = {{ $value | printf "%.2f" }}\n  LABELS = {{ $labels }}
+
+  # Alert for host mem pressure - high instantanoues mem waiting time
+  - alert: HostHighMemoryWaitingTime
+    expr: irate(node_pressure_memory_waiting_seconds_total[5m]) > 0.5
+    for: 5m
```

Comment:
Setting it to `5m` here means that not only does the rate over the last five minutes have to exceed 0.5, but this also needs to remain true for the coming five minutes before the alert actually fires. Is this intentional?

Reply:
Yes, those alert rules were set `for: 5m` because the expressions are quite easy to reach (tested with `stress`). And although they are reached, the host system can still be fairly usable, so setting `5m` can reduce many false alarms.